test/unit/entity/user.test.js

```diff
@@ -211,6 +211,30 @@ describe('User', function() {
     });
   });
 
+  describe('logout()', function() {
```

Comment:
@thomasconner the `logout()` tests should assert more that just the active user API. Please add some asserts for the cache being cleaned, sync queue emptied and checks for any other local storage items we remove (e.g. device tokens, SQL tables etc)